src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableEnumerable.DebuggerProxy.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace System.Collections.Immutable
+{
+    /// <summary>
+    /// Displays immutable enumerables in the debugger.
+    /// </summary>
+    /// <typeparam name="T">The element type of the enumerable.</typeparam>
+    /// <remarks>
+    /// This class should only be used with immutable enumerables, since it
+    /// caches the enumerable into an array for display in the debugger.
+    /// </remarks>
+    internal sealed class ImmutableEnumerableDebuggerProxy<T>
```

Comment:
I think he is just following how the `DebuggerProxy` files are now called in the source, following the same guidelines I guess. I see the names of the files he is deleting and they have the extra period in the name.